multicluster/controllers/multicluster/resourceexport_controller.go

```diff
@@ -47,16 +73,317 @@ type ResourceExportReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *ResourceExportReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	klog.InfoS("reconciling ResourceExport", "resourceexport", req.NamespacedName)
+	var resExport mcsv1alpha1.ResourceExport
+	if err := r.Client.Get(ctx, req.NamespacedName, &resExport); err != nil {
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	resImportName := types.NamespacedName{
+		Name:      getResourceImportName(&resExport),
+		Namespace: req.Namespace,
```

Comment:
req.Namespace is not equal to resExport.Namespace? If yes, I feel we need not this filed and need not pass it all funcs?

Reply:
fixed, original purpose is to reduce the call to `getResourceImportName`. move it to each func now.